zookeeper-server/src/main/java/org/apache/zookeeper/server/persistence/FileTxnSnapLog.java

```diff
@@ -102,6 +107,9 @@ public FileTxnSnapLog(File dataDir, File snapDir) throws IOException {
         boolean enableAutocreate = Boolean.parseBoolean(
             System.getProperty(ZOOKEEPER_DATADIR_AUTOCREATE, ZOOKEEPER_DATADIR_AUTOCREATE_DEFAULT));
 
+        trustEmptySnapshot = Boolean.getBoolean(ZOOKEEPER_SNAPSHOT_TRUST_EMPTY);
+        LOG.info(ZOOKEEPER_SNAPSHOT_TRUST_EMPTY + " : " + trustEmptySnapshot);
```

Comment:
nit on the log format

Reply:
What would you recommend? I thought about `        LOG.info(ZOOKEEPER_SNAPSHOT_TRUST_EMPTY + " :  {}", trustEmptySnapshot);
`, but there is a `+` I can't get rid of, so why bother :)

Reply:
I was thinking:

`LOG.info("{}:{}", ZOOKEEPER_SNAPSHOT_TRUST_EMPTY, trustEmptySnapshot);`

but just a nit